src/watts/plugin_sas.py

```diff
@@ -185,12 +186,10 @@ def postrun(self, params: Parameters) -> ResultsSAS:
         # Convert CHANNEl.dat and PRIMER4.dat to csv files
         # using SAS utilities. Check if files exist because
         # they may not be outputted per user's choice.
-        # Doesn't work with run_proc(). Use os.system() for now.
-        # Needs to convert PosixPath back to string.
         if Path("CHANNEL.dat").is_file():
-            os.system(str(self.conv_channel) + " <CHANNEL.dat> CHANNEL.csv") 
+            subprocess.run(str(self.conv_channel) + " <CHANNEL.dat> CHANNEL.csv", shell=True)
```

Comment:
@paulromano I changed this to `subprocess.run()` as you requested. Initially I tried passing the arguments as a list but that didn't work as intended. The executable was able to run but somehow wasn't able to locate the "CHANNEL.dat" file. I ended up having to pass a string with `shell=True`. I tested this on macOS, Linux, Windows and it seemed to work but I am not entirely sure if this would cause other problems. Please let me know if you have any suggestions for this.

Reply:
The "proper" way to do this with `subprocess` is to use the `stdin` and stdout` arguments:
```Python
with open("CHANNEL.dat", "r") as file_in, open("CHANNEL.csv", "w") as file_out:
    subprocess.run(str(self.conv_channel), stdin=file_in, stdout=file_out)
```